src/utils/configuration.ts

```diff
@@ -22,11 +29,12 @@ const configuration = (relativePath = '.percy.yml'): Configuration => {
     // in a chain. snapshot specific options -> agent configuration -> default values
 
     const defaultConfiguration: Configuration = {
-      version: 1.0, snapshot: {},
+      'version': 1.0,
```

Comment:
this should stay as `1.0`. These new options are completely compatible with the current configuration. We don't actually have any configuration version handling in place, this was just added for future proofing if we do find ourselves with breaking changes.